cv32e40p/tests/programs/custom/coremark/test.yaml

```diff
@@ -1,4 +1,21 @@
+# Copyright 2020 OpenHW Group
+# Copyright 2020 Silicon Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
 name: coremark
 uvm_test: uvmt_cv32e40p_firmware_test_c
+default_cflags: >
```

Comment:
looks like this is unrelated to the current issue, right?

Reply:
Correct.  This yaml did not have `ITERATIONS` defined, so the test-program did not compile.  I solved the issue by copying the `default_cflags` from the CV32E40S version of this test.  The two cores are sufficiently similar that I _believe_ this is valid.

Reply:
there is a default behavior in coremark, where it automatically determines the number of needed iterations. i think coremark requires 10k iterations or so for a valid result, which is not practical in simulation. anyway, performance has very low variance in such low end CPUs, so the result after 10 iterations is already pretty accurate. For a CVA6, this could already be different